source/backend/tests/unit/api/Services/PropertyServiceTest.cs

```diff
@@ -23,22 +23,33 @@ namespace Pims.Api.Test.Services
     [ExcludeFromCodeCoverage]
     public class PropertyServiceTest
     {
+        private TestHelper _helper;
+
+        public PropertyServiceTest()
+        {
+            _helper = new TestHelper();
+        }
+
+        private PropertyService CreateProjectServiceWithPermissions(params Permissions[] permissions)
```

Comment:
nit: please rename to appropriate class **CreatePropertyServiceWithPermissions** or what do you think of using the  same generic name on all service tests - **CreateWithPermissions** ?

I personally like the 2nd option a bit better and there's no risk of name override because the method is private - thoughts?

Reply:
No strong preference.